examples/prometheus/prometheus.yaml

```diff
@@ -118,6 +125,7 @@ objects:
           - -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
           - -cookie-secret-file=/etc/proxy/secrets/session_secret
           - -skip-auth-regex=^/metrics
+          - '-openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get", "name": "${NAMESPACE}"}}'
```

Comment:
This is a pretty low level of permission - it means anyone who can read the containing namespace can view all metrics for the entire cluster.  Another option would be to create a virtual permission that someone has to explicitly grant.  In the near term, it probably doesn't matter.